app/helpers/request_info_helper.rb

```diff
@@ -74,4 +106,13 @@ def prov_vm_grid_cells(data, edit)
     end
     cells
   end
+
+  def prov_host_grid_cells(data, edit)
+    cells = []
+    edit[:host_columns].each do |col|
+      cells.push(cell_data(h(data.send(col))))
+    end
+    cells
```

Comment:
This pattern is better presented with `.map`

```suggestion
    edit[:host_columns].map do |col|
      cell_data(h(data.send(col)))
    end
```